README.md

```diff
@@ -1,37 +1,16 @@
-# p5-template
+# Line_Brush
+-----
+This project is a cool brush following your mouse everywhere. 
 
-Use this template for a brand new p5.js project. 
+**To Run**
+-----
+In termianl -> open index.html
```

Comment:
Small note: in markdown, you can use the backtick ( ` ) to show code snippets. For example, 

`open index.html` 

How it looks in markdown:

![image](https://user-images.githubusercontent.com/26272095/124828955-d2048a00-df2c-11eb-8ad4-75dcd09d9025.png)